source/features/bypass-checks.tsx

```diff
@@ -14,8 +14,10 @@ async function bypass(detailsLink: HTMLAnchorElement): Promise<void> {
 		return;
 	}
 
-	const directLink = await api.v3(`check-runs/${runId}`);
-	detailsLink.href = directLink.details_url;
+	const {details_url: detailsUrl} = await api.v3(`check-runs/${runId}`);
+	if (detailsUrl && !detailsUrl.startsWith('https://github.com/') && new URL(detailsUrl).pathname !== '/') { // Ignore links to GitHub repos or static product pages #3938
```

Comment:
I think we can safely ignore all "details" links pointing to GitHub, because those mean an app without its own web interface.